01_prepare_host.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 if [[ $OS == ubuntu ]]; then
   sudo apt-get update
-  sudo apt -y install python3-pip jq curl wget
+  sudo apt -y install python3-pip jq curl wget pkg-config
 
   # Set update-alternatives to python3
   if [[ ${DISTRO} == "ubuntu18" ]]; then
```

Comment:
for ubuntu jammy we need to reload the correct kvm module based on brand (amd or intel), not just kvm_intel
I guess we could check cpuinfo for the virtualization feature, vmx on intel, svm for amd, and decide based on that

Reply:
I agree, but we're also in a nested VM situation, and I think at least some VMs don't have hardware-assisted nested virtualization...

Reply:
right, then we need to remove the reload in case we don't have kvm at all

Reply:
this should avoid issues with nested virtualization or in general when we don't have kvm  https://github.com/dtantsur/metal3-dev-env/pull/1